src/main/java/games/strategy/engine/framework/GameDataManager.java

```diff
@@ -140,84 +137,33 @@ private static GameData loadGameInSerializationFormat(
           System.out.println(message);
           return null;
         }
-        final String error = "<html>Incompatible engine versions, and no old engine found. We are: "
+        final String error = "Incompatible engine versions. We are: "
             + ClientContext.engineVersion() + " . Trying to load game created with: " + readVersion
-            + "<br>To download the latest version of TripleA, Please visit "
-            + UrlConstants.LATEST_GAME_DOWNLOAD_WEBSITE + "</html>";
-        if (savegamePath == null) {
```

Comment:
The `savegamePath` parameter appears to no longer be used after these changes.  Would you mind removing this parameter and pulling the thread to see how far it will take you?  It looks like 5-6 other methods will be affected by removing it but nothing too drastic.